zebra/zebra_rib.c

```diff
@@ -1293,6 +1303,44 @@ static void zebra_rib_fixup_system(struct route_node *rn)
 	}
 }
 
+static bool rib_compare_routes(const struct route_entry *re1,
+			       const struct route_entry *re2)
+{
+	bool result = false;
+
+	if (re1->type != re2->type)
+		goto done;
```

Comment:
can't we just use return result?  why do we need goto's?  I'm not a fan of goto's to avoid having more than one return statement.  If we are done return.

Reply:
sure - pushed an update.